composer.lock

```diff
@@ -4,251 +4,1165 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
         "This file is @generated automatically"
     ],
-    "content-hash": "c9216234e474339c950da59f72180d0a",
+    "content-hash": "8402beff2b9f4fd299973a3774e5ead6",
     "packages": [
         {
             "name": "lexide/k-switch",
-            "version": "1.0.1",
+            "version": "2.0.0",
+            "source": {
+                "type": "git",
+                "url": "https://gitlab.ins.risk.regn.net/UKI/shared/php/packages/lexide/k-switch.git",
```

Comment:
Ah, you need to disable the private packagist repo in your composer before composer updating; this gitlab URL is not public

Reply:
How do I do this?

Reply:
Nevermind, fixed